customer_satisfaction/streamlit_app.py

```diff
@@ -0,0 +1,90 @@
+from re import S
+import streamlit as st
+from inference.predict import predict 
+from PIL import Image
+import pandas as pd 
+
+import pickle 
+def main(): 
+    st.title("End to End Customer Satisfaction Pipeline with ZenML")
+
+
+    image = Image.open('_assets/high_level_overview.png')
+    st.image(image, caption='High Level Pipeline')
+
+    st.markdown("""
+    #### Description of Features 
+    This app is designed to predict the customer satisfaction score for a given customer.
```

Comment:
As discussed, this should start with a very quick summary of the problem, then you can illustrate with the image, and then you can specify "how to use this demo". And after that you can specify the features as you've done below.

Reply:
I think I added the explanation at the top